v2/component/launcher.go

```diff
@@ -697,3 +709,21 @@ func getExecutorOutput() (*pipeline_spec.ExecutorOutput, error) {
 
 	return executorOutput, nil
 }
+
+func (l *Launcher) openBucket() (*blob.Bucket, error) {
+	if l.bucketConfig.scheme == "minio://" {
+		sess, err := session.NewSession(&aws.Config{
+			Credentials:      credentials.NewStaticCredentials("minio", "minio123", ""),
```

Comment:
As discussed, I believe we should get the secret from [mlpipeline-minio-artifact-secret](https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/third-party/minio/base/mlpipeline-minio-artifact-secret.yaml). Do you plan to do it in a separate PR or in this PR? I think both works

Reply:
Will do it in another PR